test/ui-test/testSuites/suite_onboarding/tst_statusLoginPassword/test.feature

```diff
@@ -29,7 +29,6 @@ Feature: Status Desktop login
             | username 		   | password          |
             | Athletic_Prime   | TesTEr16843/!@00  |
 
-	@mayfail
```

Comment:
This test fails for me locally. I think that it is due to a timing issue since it is still in the loading screen when the test ends, and it doesn't waits until the icorrect password message is displayed. 
I suggest to check how / how much it waits for and maybe we need to add some validation related to the loading screen like.. if loading screen is shown, wait until it is not displayed.. then verify the error message is shown.

See the video for better understanding:

https://user-images.githubusercontent.com/97019400/231958453-d96f5c20-7a8a-4bf5-8832-4af7468a85fb.mov

Reply:
Fixed. @noeliaSD please rerun the test, the app rebuild needs